### PR TITLE
BZ#1846850 Installing private clusters requires manual install-config creation

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1846850

### CODE REVIEW

**Comment by codyhoag** [comment]
@yunjiang29 can you review? I added the manual steps for AWS, Azure, and GCP, since those are the three supported private cluster providers. Thanks!

Previews:
- AWS: https://aws-private-manual-install-config--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-private.html#installation-initializing-manual_installing-aws-private
- Azure: https://aws-private-manual-install-config--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-private.html#installation-initializing-manual_installing-azure-private
- GCP: https://aws-private-manual-install-config--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-private.html#installation-initializing-manual_installing-gcp-private

**Comment by yunjiang29** [comment]
@codyhoag LGTM.

File: modules/installation-initializing-manual.adoc [hunk 1]
```diff
@@ -37,6 +49,9 @@ ifdef::azure-gov[]
 When installing {product-title} on Microsoft Azure into a government region, you
 must manually generate your installation configuration file.
 endif::azure-gov[]
+ifdef::aws-private,azure-private,gcp-private[]
+For installations of a private {product-title} cluster that are only accessible from an internal network and are not visible to the internet, you must manually generate your installation configuration file.
```
Comment (adellape) [inline modules/installation-initializing-manual.adoc:53]: s/internet/Internet/

per IBMSG, tho looks like usage is flip-flopping in the wider, built assemblies.

**Review by adellape — COMMENTED** [review]
Minor nit, but LGTM!

File: modules/installation-initializing-manual.adoc (hunk 1 above)
Comment (codyhoag) [reply modules/installation-initializing-manual.adoc:53]: Yes, I've seen it both ways. Based on a recent peer review, I've been lower-casing. I did not know this was stated in the IBMSG :sweat_smile: . I will upper-case from now on! Great feedback; thanks!

**Pull request merged**